controllers/user.js

```diff
@@ -50,4 +51,45 @@ module.exports = {
       res.status(err.status || 500).send(err.message || "Something went wrong");
     }
   },
+
+  //////////////////////////////////
+
+  verify: async (req, res) => {
+    try {
+      //extracting email from request body
+      const { email, otp } = req.body;
+      if (isEmpty(email) || isEmpty(otp)) {
+        throw { status: 400, message: "Required fields can't be empty" };
+      }
+      //Selecting user
+      const User = await users.findOne({ where: { email: email } });
+      //Getting and matching otp in user_otps table
+      const userOTP = await user_otps.findOne({
+        where: { fk_user_id: User.id },
+        order: [["createdAt", "desc"]],
+      });
+      const currentTimestamp = moment().unix();
+      //checking if opt is not expired
+      if (userOTP.expiry < currentTimestamp) {
+        throw { status: 500, message: "OTP expired signup again" };
+      }
+      //updating user to verified
+      await User.update({ verified: 1 });
+      //Generating token...
+      const token = jwt.sign(
+        { message: "Verified" },
```

Comment:
```suggestion
        { User },
```
@Behramkhann , please encode the whole user object in the token as you will need it later.